packages/mwp-toaster/src/ToastContainer.jsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * 1. Creating toasts should _always_ be a Redux action, e.g. TOAST_MAKE
+ * 2. TOAST_MAKE will go through the platform reducer to populate an array in {{state.toast.ready}}
+ * 3. When the ToastMaster is instantiated, it will consume {{state.toast.ready}} and dispatch a new action TOAST_SHOW
+ * 4. TOAST_SHOW will _clear_ {{state.toast.ready}}
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import withRouter from 'react-router-dom/withRouter';
+import Toaster from 'meetup-web-components/lib/Toaster';
+import Toast from 'meetup-web-components/lib/Toast';
+import { makeToast, showToasts } from './actions';
+import { getReadyToasts } from './selectors';
+
+const mapStateToProps = state => ({ ready: getReadyToasts(state) });
+const mapDispatchToProps = { makeToast, showToasts };
+
+/*
+ * This component does *not* support server-rendered toasts. It must mount before
+ * toasts are added to application state through `makeToast`.
+ */
+export class ToastContainer extends React.Component {
```

Comment:
shouldn't this be PureComponent if you are using `shouldComponentUpdate`

Reply:
`PureComponent` is basically a shortcut method of implementing a [simple `shouldComponentUpdate` automatically](https://medium.com/@kylpo/all-about-reacts-shouldcomponentupdate-cc3b1e497e97#ad24) - I want to manually specify it for this component. Basically, if you are specifying your own `shouldComponentUpdate`, you probably should _not_ use `PureComponent`